internal/schema/schemas/draft/posit-publishing-schema-v3.json

```diff
@@ -115,17 +114,15 @@
         "package_manager": {
           "type": "string",
           "default": "pip",
-          "enum": ["pip", "conda", "pipenv", "poetry", "none"],
           "description": "Package manager that will install the dependencies. If package-manager is none, dependencies are assumed to be pre-installed on the server. The default is 'pip'.",
-          "examples": ["pip"]
+          "examples": ["pip", "conda", "pipenv", "poetry", "none"]
```

Comment:
Does `uv pip` work?

Reply:
No, not at this time. That is an outstanding issue (https://github.com/posit-dev/publisher/issues/2510) which uses the requirements for adding UV support as the example. For now, it won't work.